blog/2023-11-27-magazine-006.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Inflight Magazine no. 6
+description: Updates from the winglang project
+authors: 
+  - revitalb
+  - uric
+tags: [cloud-oriented programming, winglang, changelog, newsletter,]
+hide_table_of_contents: true
+---
+> The 6th issue of the Wing Inflight Magazine.
+> <!--truncate-->
+
+Hey folks!
+
+We’re back with another issue of the **Wing Inflight Magazine** and we are excited to share some updates about [winglang](https://winglang.io).
+
+- 🚀 [New: bring ui](#bring-ui)
+- 🚀 [New: Singleton](#singleton)
+- 🔥 [Remove $ from String Interpolations](#remove_dollar_from_string_interpolations)
+- 🚀 [New: Expose AWS Resources](#aws-resources)
+- 🚀 [New: Sourcemaps and Stacktraces](#sourcemaps-and-stacktraces)
+- 📅 [Upcoming Events](#upcoming-events)
+- 🎥 [The Wingly Update](#the-wingly-update)
+
+### bring ui
+
+Introducing the ui module to the SDK, a collection of APIs for adding visual interactions for your classes in the Wing Console. UI components aren't displayed in the Wing Console quite yet, but they can be accessed and called through the simulator API!
+
+![bring_us](./assets/2023-11-27-magazine-006/bring_ui.png)
```

Comment:
if this image contains code, please use a markdown codeblock instead (can use chatGPT)